tests/Http/Register/CreateTest.php

```diff
@@ -320,7 +320,7 @@
     $response = $this->from('/register')->post('/register', [
         'name' => 'Test User',
         'username' => 'testuser',
-        'email' => 'test@example.com',
+        'email' => fake()->unique()->safeEmail(),
```

Comment:
This can even generate free email as it's random in nature so please keep example.com if your goal is to test unsupported emails.

Reply:
That is the goal.